workload-team/cluster-stamp.bicep

```
@@ -161,6 +161,71 @@ resource targetVirtualNetwork 'Microsoft.Network/virtualNetworks@2023-11-01' exi
 
 /*** RESOURCES ***/
 
+// An Azure Monitor workspace where cluster metrics related to Prometheus are collected
+resource amw 'Microsoft.Monitor/accounts@2023-04-03' = {
+  name: 'amw-${clusterName}'
+  location: location
```

Just to be explicit about the network perimeter configuration.

```suggestion
  location: location
  properties: {
     publicNetworkAccess: 'Enabled'
  }
```